.github/workflows/ci.yml

```diff
@@ -10,20 +10,43 @@ on:
   pull_request:  # for all PRs regardless of its base branch
 
 jobs:
-  build:
+  build-test:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [10.x, 12.x, 14.x]
+    env:
+      CHAIN_ID: testing
+      MNEMONIC: ${{ secrets.TEST_MNEMONIC }}
 
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
+
+    - name: Use Node.js 14.x
       uses: actions/setup-node@v1
       with:
-        node-version: ${{ matrix.node-version }}
-    - run: yarn install
-    - run: yarn build
-    - run: yarn test  # TODO: Run panacead and set PANACEAD_ENABLED=true
+        node-version: 14.x
+
+    - name: Install dependencies
+      run: yarn install
+
+    - name: Build
+      run: yarn build
+
+    - name: Run a panacea-core docker container
+      run: |
+        docker run --rm -d \
+          -e CHAIN_ID="${CHAIN_ID}" \
+          -e MNEMONIC="${MNEMONIC}" \
+          -p 26657:26657 \
+          -v $(pwd)/scripts:/root/scripts \
+          --name core \
+          ghcr.io/medibloc/panacea-core:master \
+          bash /root/scripts/panacea-core/init.sh
```

Comment:
I think it's a good way.
I'll have to modify the `panacea-java` similarly later.
Thank you!